artifacts/reader.py

```diff
@@ -224,6 +208,57 @@ def _ReadSupportedOS(self, definition, definition_object, name):
 
     definition_object.supported_os = supported_os
 
+  def _ReadSources(self, artifact_definition, artifact_object):
+    """Reads the artifact definition sources
+
+    Args:
+      artifact_definition: the artifact definition as a dict.
```

Comment:
Per https://github.com/ForensicArtifacts/artifacts/pull/192 don't overload the terminology